athena-kafka/src/main/java/com/amazonaws/athena/connectors/kafka/KafkaRecordHandler.java

```diff
@@ -219,4 +251,90 @@ private void execute(
             return 1;
         });
     }
+
+    private void avroConsume(
+            BlockSpiller spiller,
+            ReadRecordsRequest recordsRequest,
+            QueryStatusChecker queryStatusChecker,
+            SplitParameters splitParameters,
+            Consumer<String, GenericRecord> kafkaAvroConsumer)
+    {
+        LOGGER.info("[kafka] {} Polling for data", splitParameters);
+        int emptyResultFoundCount = 0;
+        try {
```

Comment:
It seems we are only using a try for the sake of closing [Consumer](https://kafka.apache.org/31/javadoc/org/apache/kafka/clients/consumer/Consumer.html) -- why not use [try with resource](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)?

Reply:
Addressed